### DISCUSSION TITLE
MixxxPiBeta64v4

### DISCUSSION THREAD

**Discussion by fayaaz** [fayaaz/mixxx-pi-gen#68] [General]
## What's Changed
* Update to Debian 12 "bookworm"
* Mixxx 2.4 now in beta, built from the repo
* Install the pioneered skin
* Add  timedatectl sync  (https://github.com/fayaaz/mixxx-pi-gen/issues/55)
* Add fonts and unclutter

This release is in preparation for use on a raspberry pi 5. 
Due to not owning one, as of yet untested on the pi5.

**Full Changelog**: https://github.com/fayaaz/mixxx-pi-gen/compare/v0.7.0...v0.8.0

<hr /><em>This discussion was created from the release <a href='https://github.com/fayaaz/mixxx-pi-gen/releases/tag/v0.8.0'>MixxxPi64v4</a>.</em>

**Comment by fayaaz** [comment]
If anyone with a Raspberry Pi 5 could test this image that would be grand!

**Reply by marcouillerm** [reply]
I tested your image on my Pi 5. Boots up to mixxx directly. Only problems, can't open anything else to setup the Wifi etc.

**Reply by fayaaz** [reply]
You can use `nmtui` to set up wifi

Use Win+Enter to open a new terminal and type `nmtui` - this should give you the options to set up wifi.

**Comment by marcouillerm** [comment]
Thanks. I'm already set up on the raspberry pi5 and mixxx 2.4 + Karaoke
machine. Things are running well. Tried Mixxx with my controller and that's
super smooth.

Le jeu. 22 févr. 2024 à 20:07, Fayaaz Ahmed ***@***.***> a
écrit :
